lib/tasks/find_possible_duplicate_accounts.rake

```diff
@@ -0,0 +1,93 @@
+namespace :accounts do
+  namespace :duplicate_accounts do
+    desc "Find possible duplicate acccounts"
+    task :find => [:environment] do
+      CSV.open("duplicate_users_by_name.csv", 'w+') do |csv|
+        csv << ['User First Name',
+                'User Last Name',
+                'Email Address(es)',
+                'User ID',
+                'Applications',
+                'Social Signup Successful?',
+                'Reset Password Help Requested?',
+                'Help Request Failed?',
+                'Authentication Transfer Failed?'
+              ]
+
+        where_names_match = User.joins{ User.unscoped.as(same)
+                        .on{
+                          (same.id != ~id) & (lower(same.first_name) == lower(~first_name)) & (lower(same.last_name) == lower(~last_name))
+                        }
+                      }.uniq.order(id: :asc)
```

Comment:
want to order based on first and last name so potential dupes are near each other?